src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        int index = new Random().nextInt(Color.values().length);
+        return Color.values()[index].toString();
     }
```

Comment:
Avoid creating a new instance of Random each time the method is called. Instead, create a single instance as a class-level variable. Also, use the name() method instead of toString() to get the String representation of enum constants.